CONTRIBUTING.md

```diff
@@ -1,49 +1,151 @@
+- If you find the issue and it's free, assign yourself to it or leave a comment to notify people you're going to start the translation.
+- If the file you wish to translate is missing from the issues list (which can happen), open a new issue using the template "File to translate".
+- Translate the page entirely and submit it for review before starting another page.
 
-## Making Your First Translation PR
+### 2. Clone the repository / Keep your fork updated
 
-- Fork this repository (click "fork" on the repository's home page in GitHub)
+If it's your first contribution, fork this repository from GitHub interface. Then on your machine:
+```
+git clone <your fork's url>
+git remote add upstream https://github.com/dazzlingfugu/ember-fr-guides-source.git
+```
+
+As you tackle new issues, you'll want to be sure that you always start by working on the most recent code. The second command is to set an upstream. By pulling from it, you can sync up your fork's `master` with a parent repository's `master`. For this to work, you should make sure you're always committing to a branch, not `master`.
+
+Each time you want to start a new page:
+```
+git checkout master
+git pull upstream master
+git checkout -b some-branch-name
+```
+
+### 3. Translate and lint the page
 
-- Clone the forked repository with `git clone <your fork's url>` and create a branch with `git checkout -b some-branch-name`.
+Each PR should have:
+- 1 commit
+- 2~3 files changed (`[some-page].md`, `.remarkignore`, optionally `.local.dic`)
 
-- Keep your commit stack clean with one commit, for instance: `git commit -m "feat: translate guides/release/index.md`
+While you are working, keep your commit stack clean with one commit, for instance: `git commit -m "feat: translate guides/release/index.md`.
 
-- Once you're at the point that you'd like feedback, submit a Pull Request (new Pull Request button). Choose `master` for the base and your branch name for `compare`, then submit it!
+Once you have translated the `.md` page of your choice, you need to run the linter to make sure there's no typo:
+- Add the path to your page in `.remarkignore` (respect the alphabetical order for readability)
+- Run `npm run lint`
 
-- Your PR will be reviewed by another contributor, and then either merged or have changes requested.
+If there are errors because "a word is written twice", but it's expected in French (Je vais vous "faire faire" une jolie traduction), make sure to fix all the other errors of the same paragraph before adding `<!-- spell ignore -->` above the paragraph.
 
-You are free to use the method you like to translate the file. Here are a few recommendations:
+If there are errors because you kept an English term commonly used in French, add this word the the `.local.dic` file (respect the alphabetical order for readability, lowercase words are below uppercase acronyms).
```

Comment:
```suggestion
If there are errors because you kept an English term commonly used in French, add this word to the `.local.dic` file (respect the alphabetical order for readability, lowercase words are below uppercase acronyms).
```